lib/prosperworks/loss_reason.rb

```diff
@@ -0,0 +1,12 @@
+module ProsperWorks
+  class LossReason < BaseEntity
```

Comment:
This should inherit from `Base` and not `BaseEntity`. 

I added in some comments to both classes hopefully providing a little more context, and when to use one or the other.